fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/zh_cn.json

```diff
@@ -0,0 +1,3 @@
+{
+  "pack.source.fabricmod": "Fabric模组"
```

Comment:
Why no space here while space is present between character and words in other languages?

Reply:
I'm just following the vanilla convention. In vanilla assets files, `zh-cn.json` usually does not leave spaces between character and words, while `zh-hk.json` and `zh_tw.json` do.